src/certifier.mak

```diff
@@ -84,11 +83,13 @@ SWIG_CERT_INTERFACE = certifier_framework
 # Base of SWIG-Pytest's interface file for use by SWIG
 SWIG_PYTEST_INTERFACE = swigpytests
 
+FIX_SWIG_SCRIPT = $(CERTIFIER_ROOT)/CI/scripts/fix_swig_wrap.sh
+
 PY_INCLUDE = -I /usr/include/python3.10/
 
 #export LD_LIBRARY_PATH=/usr/local/lib
 LDFLAGS = -L $(LOCAL_LIB) -lprotobuf -lgtest -lgflags -lpthread -L/usr/local/opt/openssl@1.1/lib/ -lcrypto -lssl
-LDFLAGS_SWIGPYTEST = -L $(LOCAL_LIB)
+LDFLAGS_SWIGPYTEST = -L $(LOCAL_LIB) -l protobuf
```

Comment:
Some SWIG-unit-tests now use protobuf classes in the interfaces, hence need to link with protobuf libraries while building `swigpytestssharedlib` library.